jetty9-base/src/main/jetty-base/config-scripts/jetty.commands

```diff
@@ -0,0 +1,11 @@
+--create-startd
+
+--add-to-start=server,webapp,http,deploy,jsp,jstl,resources
+
+jetty.httpConfig.outputAggregationSize=32768
```

Comment:
Do these settings vary from Jetty defaults? Should they be part of the gcp module?